firstvoices/backend/models/media.py

```diff
@@ -180,8 +180,13 @@ def save(self, update_metadata_command=False, **kwargs):
             with get_local_video_file(self.content) as temp_file:
                 video_info = self.get_video_info(temp_file)
 
-            self.width = int(video_info["width"])
-            self.height = int(video_info["height"])
+            if video_info is None:
+                self.logger.error(
```

Comment:
I think this could be a warning, since it is worth looking into but doesn't interrupt service on the website.